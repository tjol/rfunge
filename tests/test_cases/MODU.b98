"PXIF"4#@("UDOM"4#@(a3/" = 3 / 01"8k,." MER"3k,a3%.a,v
                   v ,a.R3a,,"R ".U3a,,"U ".M3a,,"M "<
                   >a3N/" = 3- / 01"9k,." MER"3k,a3N%.a,v
                   v ,a.RN3a,,"R ".UN3a,,"U ".MN3a,,"M "<
                   >aN3/" = 3 / 01-"9k,." MER"3k,aN3%.a,v
                   v ,a.R3Na,,"R ".U3Na,,"U ".M3Na,,"M "<
                   >aN3N/" = 3- / 01-"ak,." MER"3k,aN3N%.a,v
                   @ ,a.RN3Na,,"R ".UN3Na,,"U ".MN3Na,,"M "<