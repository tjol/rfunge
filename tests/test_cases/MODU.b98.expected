10 / 3 = 3 REM 1 
M 1 U 1 R 1 
10 / -3 = -3 REM 1 
M -2 U 1 R 1 
-10 / 3 = -3 REM -1 
M 2 U 2 R -1 
-10 / -3 = 3 REM -1 
M -1 U 2 R -1 
